tests/sentry/data_export/endpoints/test_data_export.py

```diff
@@ -17,6 +17,17 @@ def setUp(self):
         self.organization = self.create_organization(owner=self.user)
         self.login_as(user=self.user)
 
+    def test_authorization(self):
```

Comment:
Worth testing project perms here as well.

Reply:
Will add to https://github.com/getsentry/sentry/pull/18229!